control/HTTP.php

```diff
@@ -36,15 +36,42 @@ class HTTP {
 	 */
 	private static $disable_http_cache = false;
 
+	/**
+	 * Mapping of extension to mime types
+	 *
+	 * @var array
+	 * @config
+	 */
+	private static $MimeTypes = array();
```

Comment:
Why the uppercase `M`?

Reply:
Because that's the casing used in YML since dawn of time. It's been missing from PHP for ages though.